src/frontend/components/node-item/node-attributes.ts

```diff
@@ -9,4 +9,11 @@ import {Property} from '../../../backend/utils';
 })
 export class NodeAttributes {
   @Input() private attributes: Array<Property>;
+
+  private show: boolean = false;
+
+  onShow(event) {
+    event.stopPropagation();
```

Comment:
This part has actually been removed, there is no click event to listen for anymore